scenario_09_sandbox.lua

```diff
@@ -6013,6 +6013,7 @@ function icarusSector()
 end
 function createIcarusColor()
 	icarus_color = true
+	icarusWormholeRiptideStuff = {}
```

Comment:
I typically put these things in their own spawn routine. For example, there's the worm hole tour which is not always spawned with Icarus. Similarly, the asteroid whirlpool in the Kentar region can at least be despawned. I thought I had made the riptide terrain independent of Icarus even though there is a link near Speculator. Maybe that was just the destroy routine.

All this leads to the question: how much testing did you do?

Reply:
Did a lot of changes, testing despawning must've slipped my mind, sorry